README.md

```diff
@@ -130,6 +130,28 @@ module.exports = [
 ];
 ```
 
+Add this field to any desired content-type attributes
+
+```json5
+{
+	// ...
+	"attributes": {
+		// ...
+		"video": {
+			"type": "customField",
+			"customField": "plugin::video-field.video",
+			"default": "{}"
+		}
+		// ...
+	}
+}
+```
+
+> **Note**  
+> The `"default": "{}"` seems to be needed in latest versions of Strapi, otherwise Admin will crash at content-type creation
```

Comment:
Maybe we can fix it in plugin, and try/catch JSON.parse?

Reply:
Thanks! This custom field was causing errors in my Strapi instance and this solved it.